nxc/modules/add-computer.py

```diff
@@ -178,7 +177,6 @@ def do_samr_add(self, context):
                     samr.hSamrLookupNamesInDomain(dce, domain_handle, [self.__computerName])
                     self.noLDAPRequired = True
                     context.log.highlight("{}".format('Computer account already exists with the name: "' + self.__computerName + '"'))
-                    sys.exit(1)
```

Comment:
This one should exit as well imo. Otherwise it's still trying to add the computer

Reply:
when you are already on it you could change the line above to a proper format string as well :D

Reply:
I don't think we should exit(1) for every single error. The program ran fine, but the computer account already exists.

Reply:
Sounds good if you tested it